tt_eager/tt_lib/__init__.py

```diff
@@ -1,5 +1,79 @@
 # SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
 
 # SPDX-License-Identifier: Apache-2.0
+import os
+import subprocess
+from pathlib import Path
+from loguru import logger
 
+
+def _has_not_found(target_so):
+    if not os.path.exists(target_so):
+        logger.trace(f"Shared library {target_so} not exists")
```

Comment:
Please change `not exists` to `does not exist`

Reply:
ok, I have fixed it